server/cli/launch.py

```diff
@@ -26,14 +26,14 @@
 
 def annotation_args(func):
     @click.option(
-        "--experimental-annotations",
+        "--disable-annotations",
         is_flag=True,
         default=False,
         show_default=True,
-        help="Enable user annotation of data.",
+        help="Disable user annotation of data.",
     )
     @click.option(
-        "--experimental-annotations-file",
+        "--annotations-input-file",
```

Comment:
The name is misleading - it is both the input and output file.  It will be overwritten.

I think the original name `--annotations-file` is clearer.  And definitely more accurate.

CC: @liaprins-czi

Reply:
@liaprins-czi -- I'll defer to you on these two CLI args. Just let me know what you'd like and I'll update.

Reply:
I think Bruce's rationale for `--annotations-file` makes a lot of sense, and it is shorter too, so I would go with that.